server/test/factories.ts

```diff
@@ -179,6 +179,7 @@ export async function buildInvite(overrides: Partial<User> = {}) {
     name: `User ${count}`,
     createdAt: new Date("2018-01-01T00:00:00.000Z"),
     invitedById: actor.id,
+    authentications: [],
```

Comment:
it seems correct to blank out authentications to make it as close to a "real" invite as possible